install/deploy/base/rhods-application.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: applications.console.openshift.io/v1alpha1
+kind: RhodsApplication
+metadata:
+  name: jupyterhub
+spec:
+  displayName: JupyterHub
+  provider: Jupyter
+  description: |-
+    A multi-user version of the notebook designed for companies, classrooms and research labs.
+  kfdefApplications: ['jupyterhub', 'notebook-images']
+  csvName: opendatahub
+  route: jupyterhub
+  routeNamespace: redhat-ods-applications
+  img: images/jupyterhub.svg
+  category: Red Hat managed
+  support: red hat
+  docsLink: https://jupyter.org/hub
+  quickStart: create-jupyter-notebook
+  getStartedLink: https://jupyterhub.readthedocs.io/en/stable/getting-started/index.html
+---
+apiVersion: applications.console.openshift.io/v1alpha1
+kind: RhodsApplication
+metadata:
+  name: aikit
+spec:
+  displayName: AIKit
+  provider: Intel
+  description: |-
+    A conda based notebook with intel-aikit packages installed.
+  kfdefApplications: ['jupyterhub', 'notebook-images']
+  csvName: opendatahub
+  route: jupyterhub
+  routeNamespace: redhat-ods-applications
+  img: images/intel.svg
+  category: Intel managed
+  support: Intel
+  docsLink: https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html
+  quickStart: create-aikit-notebook
+  getStartedLink: https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top.html
```

Comment:
We need to add all the applications from the current `data/applications` right?

Reply:
yes, i'll update